card-flow.md

```diff
@@ -0,0 +1,52 @@
+# Project Management Boards
+
+Our team uses project boards to manage our projects and track our progress.  The boards serve two purposes: to allow clients to see what's happening and give feedback on a project, and to allow us to manage our team, our tasks, and stay coordinated. Each project has its own board. The board shows us the current state of all the things we're working on in our project and where each task is in the lifecycle of development. Each task is represented by a card on the board, and as cards move through the development process, we move them to different columns on the board.
+
+Everything the team is working on for a project, whether a feature, a bug, or a chore, should be represented by a card. We can't track something if it's not on the board. If we're doing something that's not represented by a card, we should ask why.
+
+## Board Flow
+
+Our current project board tool is not very expressive in some ways, and this leads us to add additional columns to our board. Currently, we have a "Backlog" column in our project board. Once Feature Complete is up and running, our backlog will live in the Features area of the app rather than on the project board itself. The "Backlog" column is where we store tasks that will need to be completed but it's either not the right time to begin work on them or they haven't yet been approved.
+
+#### The "Next" Column
```

Comment:
I'm curious why you went from 2 #s to 4 #s here?  I would assume 3 #s would be used here.